host_collection-0.yml

```diff
@@ -0,0 +1,240 @@
+interactions:
```

Comment:
Those files are definitely in the wrong directory. Anyway, they should be automatically generated by `make record_host_collection` in which case, they end up in the right place.

Reply:
Haha, they sure are. The `patch` command didn't work quite well for me. Sorry I missed that. :)

Reply:
just don't bother copy them. Drop em here, they are generated later.

Reply:
Ok, so I should remove all 3 fixture files from the commit? Just add the playbook and task file under test then?

Reply:
Yes, drop them. They wont work with your new module anyway.
Before this pr is finished, you need to record those fixtures against a real Katello instance. Have a look at the README's How to test section.